src/jit/compiler.h

```diff
@@ -207,35 +207,44 @@ struct VarScopeDsc
 #endif
 };
 
-// This is the location of a SSA definition.
-struct DefLoc
+// This class stores information associated with a LclVar SSA definition.
+class LclSsaVarDsc
 {
-    BasicBlock* m_blk;
-    GenTree*    m_tree;
+    BasicBlock* m_block;
+    GenTreeOp*  m_asg;
```

Comment:
It does not belong to this PR but would it be possible to clean up the naming of members by getting by replacing `m_asg` and similar identifiers and start using self-documenting identifiers i.e. `m_Assignment`. IMHO we are not that much restricted by identifier naming and IMO skipping all 3 letter Verb or Noun shortcuts in the code help very much in improving code reading and understanding.